akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/javadsl/TestProbe.scala

```diff
@@ -24,7 +24,7 @@ object FishingOutcomes {
   /**
    * Consume this message and continue with the next
    */
-  def continue(): FishingOutcome = FishingOutcome.Continue
```

Comment:
Should we add a new method `continueAndCollect` and deprecated the old `continue` one?

Reply:
Let's wait for other reviewers, ok ?